pkg/knuu/instance.go

```diff
@@ -896,6 +975,70 @@ func (i *Instance) DisableNetwork() error {
 	return nil
 }
 
+// SetBandwidthLimit sets the bandwidth limit of the instance
+// bandwidth limit in bps (e.g. 1000 for 1Kbps)
+// Currently, only one of bandwidth, jitter, latency or packet loss can be set
+// This function can only be called in the state 'Commited'
+func (i *Instance) SetBandwidthLimit(limit int) error {
+	if !i.IsInState(Committed) {
+		return fmt.Errorf("setting bandwidth limit is only allowed in state 'Committed'. Current state is '%s'", i.state.String())
+	}
+	if i.isNetworkConfigSet() {
+		return fmt.Errorf("setting bandwidth limit is not allowed if network config is already set")
+	}
+	i.networkConfig.bandwidth = limit
```

Comment:
Ideally it should be a function call which calls a function from the bitTwister SDK.